example/lib/fake_chart_series.dart

```diff
@@ -61,10 +61,56 @@ class FakeChartSeries {
     return data;
   }
 
+  Map<DateTime, double> createLine2_3() {
+    Map<DateTime, double> data = {};
+
+    data[DateTime.now().subtract(Duration(minutes: 80))] = 10.0;
+    data[DateTime.now().subtract(Duration(minutes: 70))] = 15.0;
+    data[DateTime.now().subtract(Duration(minutes: 60))] = 15.0;
+    data[DateTime.now().subtract(Duration(minutes: 50))] = 25.0;
+    data[DateTime.now().subtract(Duration(minutes: 40))] = 13.0;
+    data[DateTime.now().subtract(Duration(minutes: 30))] = 24.0;
+    data[DateTime.now().subtract(Duration(minutes: 23))] = 37.0;
+    data[DateTime.now().subtract(Duration(minutes: 22))] = 39.0;
+    data[DateTime.now().subtract(Duration(minutes: 20))] = 29.0;
+    data[DateTime.now().subtract(Duration(minutes: 15))] = 27.0;
+    data[DateTime.now().subtract(Duration(minutes: 12))] = 9.0;
+    data[DateTime.now().subtract(Duration(minutes: 5))] = 35.0;
+    return data;
+  }
+
+  Map<DateTime, double> createUpperCriticalLine() {
```

Comment:
Could it be named to something more generic? For example: `yAxisUpperMarkerLine` ?